src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -112,6 +112,14 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
         // handle keyboard events in the menu
         this.menu._onKeydown$.pipe(takeUntil(this._onDestroy$))
             .subscribe(event => this.onMenuKeydown(event));
+
+        this.menu._placement$.pipe(takeUntil(this._onDestroy$))
+            .subscribe((placement) => {
+                if (!this._isSubmenuTrigger) {
+                    this.menu.placement = placement;
```

Comment:
This should not be necessary, the new value provided to `placement` is what triggers this event.